contracts/src/v0.8/functions/dev/1_0_0/RouterBase.sol

```diff
@@ -218,20 +198,19 @@ abstract contract RouterBase is IRouterBase, Pausable, ITypeAndVersion, Confirme
       address from = s_route[id];
       address to = s_proposedContractSet.to[i];
       s_route[id] = to;
-      emit ContractUpdated(id, from, to);
+      emit ContractUpdated({
+        proposedContractSetId: id,
```

Comment:
Why arguments for this event are prefixed "proposed"?

Reply:
Renamed!